src/modules/campaign-monitor/helpers/crypto-helper.ts

```diff
@@ -0,0 +1,41 @@
+import crypto from 'crypto';
```

Comment:
Q: are these helpers really needed 🤔 

@bertyhell thoughts on this approach?

Reply:
The Jira ticket had a guide on how to do the encryption/decryption which uses crypto. Crypto was already added as a package.